src/firefly/js/charts/ui/ChartsContainer.jsx

```diff
@@ -86,12 +86,19 @@ ChartsContainer.propTypes = {
 };
 
 function ExpandedView(props) {
+    const {defaultCharts, closeable} = props;
     return (
         <div style={{ display: 'flex', height: '100%', flexGrow: 1, flexDirection: 'column', overflow: 'hidden'}}>
             <div style={{marginBottom: 3}}>
-                {props.closeable && <CloseButton style={{display: 'inline-block', paddingLeft: 10}} onClick={() => dispatchSetLayoutMode(LO_MODE.expanded, LO_VIEW.none)}/>}
+                {closeable && <CloseButton style={{display: 'inline-block', paddingLeft: 10}} onClick={() => dispatchSetLayoutMode(LO_MODE.expanded, LO_VIEW.none)}/>}
             </div>
-            <ChartsTableViewPanel expandedMode={true} expandable={false} {...props} />
+            <ChartsTableViewPanel expandedMode={true} expandable={false} chartType={'scatter'} {...props} />
```

Comment:
This whole section of code is a simple cut/paste from standard view.  You should clean it up as you deemed necessary.  Although, it does seem a bit awkward to see so many components laid out, but only one get mounted.